lib/components/standform/form.tsx

```diff
@@ -0,0 +1,134 @@
+"use client";
+
+import { ForceMountTabs } from "@/lib/components/forms/force-mount-tabs";
+import {
+	Card,
+	CardContent,
+	CardHeader,
+	CardTitle,
+} from "@/lib/components/ui/card";
+import { TabsTrigger } from "@/lib/components/ui/tabs";
+import { CounterInput } from "@components/forms/counter-input";
+import { Checkbox } from "@components/ui/checkbox";
+import { Label } from "@components/ui/label";
+import { TabsList } from "@radix-ui/react-tabs";
+import { useEffect, useState } from "react";
+import { getAutoTab, getEndgameTab, getMiscTab, getTeleopTab } from "./tabs";
+import { Button } from "@/lib/components/ui/button";
+import { StandFormData, validate } from "@/lib/actions/stand-form";
+
+export function EndgameContent() {
+	const [climbed, setClimbed] = useState(false);
+	const [parked, setParked] = useState(false);
+
+	useEffect(() => {
+		if (climbed) {
+			setParked(false);
+		}
+	}, [climbed]);
+
+	useEffect(() => {
+		if (parked) {
+			setClimbed(false);
+		}
+	}, [parked]);
+
+	return (
+		<div className="space-y-2">
+			<div className="flex items-center space-x-2">
+				<Checkbox
+					id="climbed"
+					className="size-6"
+					name="climbed"
+					checked={climbed}
+					onCheckedChange={() => setClimbed((curr) => !curr)}
+				/>
+				<Label htmlFor="climbed">Climbed?</Label>
+			</div>
+			<div className="flex items-center space-x-2">
+				<Checkbox
+					id="parked"
+					className="size-6"
+					name="parked"
+					checked={parked}
+					onCheckedChange={() => setParked((curr) => !curr)}
+				/>
+				<Label htmlFor="parked">Parked?</Label>
+			</div>
+			<div hidden={!climbed}>
+				<Label htmlFor="bots_on_chain">Bots Same Chain</Label>
+				<CounterInput
+					defaultValue={1}
+					required
+					min={1}
+					max={3}
+					name="bots_on_chain"
+				/>
+			</div>
+		</div>
+	);
+}
+
+export function StandForm({
+	handleSubmit,
+}: {
+	handleSubmit: (data: StandFormData) => Promise<unknown>;
+}) {
+	const [errors, setErrors] = useState([] as string[]);
```

Comment:
Check out useFormState: https://react.dev/reference/react/useActionState

(Called useActionState in React 19, but in React 18 it's useFormState)